db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180329164621) do
+ActiveRecord::Schema.define(version: 20190223070112) do
```

Comment:
For learning: The version seems to be the date and time. Is this entered manually?
Also, is the version always changed when the schema file is changed?

Reply:
Not entered manually. And yes updated automatically.